lib/danger/request_sources/github/github.rb

```diff
@@ -375,6 +375,12 @@ def find_position_in_diff(diff_lines, message, kind)
         pattern = "+++ b/" + message.file + "\n"
         file_start = diff_lines.index(pattern)
 
+        # Files containing spaces sometimes have a trailing tab
+        if file_start.nil?
+          pattern = "+++ b/" + message.file + "\t\n"
+          file_start = diff_lines.index(pattern)
```

Comment:
I could simplify this entire block down to: 

```ruby
pattern = "+++ b/" + message.file + "\n"
pattern_with_tab = "+++ b/" + message.file + "\t\n"
file_start = diff_lines.index(pattern) || diff_lines.index(pattern_with_tab)
```

but wasn't sure about the change in readability. Thoughts?